index.html

```diff
@@ -77,19 +77,32 @@ <h2 class="h5">Vite Ma Dose est-il à but lucratif&nbsp;?</h2>
-                    <li>femmes enceintes à partir du 2e trimestre de grossesse.</li>
-                </ul>
-                <p>Les professionnels de santé et du secteur médico-social sont également éligibles à la vaccination contre la Covid-19.</p>
+                <div class="row">
+                    <div class="col col-sm-24 col-md mb-5 mb-md-0">
+                        <h2 class="h5">Qui peut se faire vacciner contre la Covid-19&nbsp;?*</h2>
+                        <p>
+                            De nombreuses personnes sont éligibles à la vaccination contre la Covid-19. On peut être éligible à la vaccination selon son âge, sa santé, ou sa profession. Voici la liste des personnes qui peuvent dès à présent être vaccinées&nbsp;:
+                        </p>
+                        <ul>
+                            <li>personnes de 55 ans et plus quel que soit leur lieu de vie et leur état de santé (avec ou sans comorbidités)&nbsp;;</li>
+                            <li>personnes de 50 à 54 ans inclus souffrant d’une pathologie à très haut risque de forme grave de Covid-19 ou d'une ou plusieurs comorbidités&nbsp;;</li>
+                            <li>personnes de 18 à 49 ans inclus souffrant d’une pathologie à très haut risque de forme grave de Covid-19&nbsp;;</li>
+                            <li>personnes majeures en situation de handicap, hébergées en maison d’accueil spécialisée ou foyer d’accueil médicalisé&nbsp;;</li>
+                            <li>adultes vivant dans le même foyer qu’une personne sévèrement immunodéprimée, enfant ou adulte (transplantés d’organes solides, transplantés récents de moelle osseuse récents, patients dialysés, patients atteints de maladies auto-immunes sous traitement immunosuppresseur fort de type anti-CD20 ou anti-métabolites).</li>
+                            <li>femmes enceintes à partir du 2e trimestre de grossesse.</li>
+                        </ul>
+                        <p>Les professionnels de santé et du secteur médico-social sont également éligibles à la vaccination contre la Covid-19.</p>
+                        <p class="fw-light fst-italic">Mis à jour le 30/04/2021</p>
+                        <p class="text-secondary fw-light fst-italic">* Ces informations sont susceptibles d'évoluer en fonction de la situation sanitaire. Tous les détails sont disponibles sur le site <a class="link-primary" href="https://solidarites-sante.gouv.fr/grands-dossiers/vaccin-covid-19/">https://solidarite-sante.gouv.fr</a>.</p>
+                    </div>
+                    <div class="col-sm-24 col-md">
+                        <div class="d-flex justify-content-md-end">
+                            <a aria-hidden="true" href="https://www.gouvernement.fr/sites/default/files/cimages/infog_vaccins_particuliers.png" target="_blank">
```

Comment:
Hello, 
On pourrait mettre le lien vers la version PDF https://solidarites-sante.gouv.fr/IMG/pdf/infog_vaccins_particuliers.pdf.
Ainsi l'attribut `aria-hidden="true"` n'est plus nécessaire.
Enfin `target="_blank"` a besoin de l'attribut `rel="noopener"`.

```suggestion
                            <a href="https://solidarites-sante.gouv.fr/IMG/pdf/infog_vaccins_particuliers.pdf" target="_blank" rel="noopener">
```

Cdt.

Reply:
Bonne idée, je n'avais pas vu le lien vers le pdf ! Je vais faire les changements.